src/domain/wallets/index.types.d.ts

```diff
@@ -52,6 +52,8 @@ type BaseWalletTransaction = {
   readonly settlementAmount: Satoshis | UsdCents
   readonly settlementFee: Satoshis | UsdCents
   readonly settlementCurrency: WalletCurrency
+  readonly settlementDisplayAmount: number
+  readonly settlementDisplayCurrency: DisplayCurrency | ""
```

Comment:
shouldn't be this `DisplayCurrency.Usd`? or just `DisplayCurrency` and guarantee this value from translation?